data/operators/landuse/industrial.json

```diff
@@ -26,6 +26,22 @@
         "operator:wikidata": "Q1053348"
       }
     },
+    {
+      "displayName": "LKQ Pick Your Part",
+      "locationSet": {"include": ["us"]},
+      "preserveTags": ["^name"],
+      "tags": {
+        "brand": "Pick Your Part",
+        "brand:short_name": "PYP",
+        "brand:wikidata": "Q133309706",
+        "industrial": "auto_wrecker",
+        "landuse": "industrial",
+        "name": "Pick Your Part",
+        "official_name": "LKQ Pick Your Part",
+        "operator": "LKQ",
+        "operator:wikidata": "Q20711490"
```

Comment:
[Pick Your Part <small>(Q133309706)</small>](https://www.wikidata.org/wiki/Q133309706) says the chain is a subsidiary of [LKQ Corporation <small>(Q20711490)</small>](https://www.wikidata.org/wiki/Q20711490). We don’t normally use `operator=*` to tag corporate hierarchies. It’s primarily for situations where the operator is orthogonal to the brand (think franchisee), which is why `operator=*` rarely occurs in the brand/ tree and vice versa. Since there is a brand involved, consider moving this entry to a new data/brands/landuse/industrial.json.

Reply:
To be clear, `operator=*` also happens to be correct tagging on these scrapyards. However, it isn’t the job of NSI to standardize both `brand=*` and `operator=*` at the same time, or even if so, `brand=*` takes precedence. Otherwise, we’d have a lot more owned-and-operated chains under the brand/ tree to add `operator=*` to.

Reply:
@1ec5 to speak to how intrusive LKQ is in the brand....

![Screenshot_2025-03-20_15-30-51](https://github.com/user-attachments/assets/7216dfe5-933f-4ef9-bd80-29987db9ccd6)

That's the kind of stuff on the building that i'm mapping

Reply:
I would love it for NSI to be more granular so that we can standardize `brand=*` and `operator=*` individually with their QIDs on a given feature. #10181 is a prime example why. But that would require significant work in this repository and upstream and downstream.

Reply:
Ah, so this isn't a taxonomy issue as much as it is an NSI shortcoming now, that i'll have to read more about. Cool cool. I'll get this moved over to brand today then. We're good.